static/js/https/domains.js

```diff
@@ -79,118 +83,168 @@ $(document).ready(function () {
 
   // Construct a sentence explaining the HTTP situation.
   var httpDetails = function(data, type, row) {
+
     if (type == "sort")
       return null;
 
     var https = row.https.uses;
     var behavior = row.https.enforces;
     var hsts = row.https.hsts;
     var hsts_age = row.https.hsts_age;
+    var preloaded = row.https.preloaded;
+    var grade = row.https.grade;
+
+    var tls = [];
+
+    // If an SSL Labs grade exists at all...
+    if (row.https.grade >= 0) {
+
+      if (row.https.sig == "SHA1withRSA")
+        tls.push("Certificate uses a " + l("sha1", "weak SHA-1 signature"));
+
+      if (row.https.ssl3 == true)
+        tls.push("Supports the " + l("ssl3", "insecure SSLv3 protocol"));
+
+      if (row.https.tls12 == false)
+        tls.push("Lacks support for the " + l("tls12", "most recent version of TLS"));
+    }
```

Comment:
I removed the information messages for `RC4` and for forward secrecy. 

RC4 is going to be shut off by browsers regardless, and the only sites that have to worry are sites which **only** use RC4, which I don't think exists in (federal) `.gov`.

Forward secrecy is important, but now that we're gating "Perfect score!" on TLS messages, I no longer think it's worth calling out specifically over it. Since we still gate on A+, SSL Labs can choose to enforce that if it wants (I don't think it holds up an A+ over it, according to their [Rating Guide](https://www.ssllabs.com/downloads/SSL_Server_Rating_Guide.pdf)).